code/datums/banking.dm

```diff
@@ -205,6 +206,22 @@
 		..()
 		interact_particle(user,src)
 
+	attackby(obj/item/W as obj, mob/user as mob)
+		if (can_reconnect)
+			if (istype(W, /obj/item/device/multitool) && !(status & (BROKEN|NOPOWER)))
+				boutput(user, "<span class='notice'>You pulse [src.name] to re-scan for equipment.</span>")
+				connection_scan()
+				return
+			else
+				src.attack_hand(user) //Previously the default behaviour for all affected computers
```

Comment:
This feels weird. What if there's a future reconnectable device which doesn't want to do attack_hand otherwise? I guess it can just handle it in its own `attackby` but still a bit ugly. Though not a big issue, it's ok to keep it as is.

Reply:
I agree it's a bit of a weird one but I also don't know if people have gotten used to being able to use these with crap in their hands. Previously I did ```attack_hand``` + ```..()``` for the computers touched but that's also a bit ugly. (It'd be functionally the same except it also gives a "uselessly hits with" message)

It seems like there's enough of a precedent for this (also with stuff like DWAINE terminals) that new computers would need good reason not to.